packages/player/src/components/shortcut.tsx

```diff
@@ -391,7 +390,12 @@ export const Shortcut: React.FC<ShortcutProps> = ({
         )
       })[0]
 
-      if (shortcut) {
+      if (
+        shortcut &&
+        shortcutsEnabled &&
+        (document.activeElement?.nodeName === 'BODY' ||
+          document.activeElement?.nodeName === 'VIDEO')
+      ) {
```

Comment:
inspired by youtube player behaviour. only listen to shortcuts if user has clicked on body or video element. and disable if anything else is interacted with. seem to be working very well.